packages/agent-toolkit/src/examples/langchain.ts

```diff
@@ -0,0 +1,86 @@
+import type { ChatPromptTemplate } from "@langchain/core/prompts";
+import { ChatOpenAI } from "@langchain/openai";
+import { config } from "dotenv";
+import { existsSync } from "fs";
+import { AgentExecutor, createStructuredChatAgent } from "langchain/agents";
+import { pull } from "langchain/hub";
+import { dirname, resolve } from "path";
+import { fileURLToPath } from "url";
+
+import { RecallAgentToolkit } from "../langchain/index.js";
```

Comment:
can we set things up so that the import is actually the public api import? I think we can do that with aliases right? so it'd be `import { RecallAgentToolkit } from "@recallnet/agent-toolkit/langchain"` here?